production/kustomization.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Why did you delete this file and configmap?

Reply:
That's weird because I don't remember deleting that